tools/firmware-utils/src/jcgimage.c

```diff
@@ -387,8 +400,8 @@ main(int argc, char **argv)
 		sizeo = sizeof(*jh) + sizeu;
 	}
 
-	if (sizeo > MAXSIZE) {
-		errx(1,"payload too large: %zd > %zd\n", sizeo, MAXSIZE);
+	if (sizeo > maxsize) {
```

Comment:
WARNING: braces {} are not necessary for single statement blocks
    #87: FILE: tools/firmware-utils/src/jcgimage.c:403:
    +       if (sizeo > maxsize) {
    +               errx(1,"payload too large: %zd > %zd\n", sizeo, maxsize);
            }